ts/Extensions/FullScreen.ts

```diff
@@ -368,3 +377,118 @@ addEvent(Chart, 'beforeRender', function (): void {
      */
     this.fullscreen = new H.Fullscreen(this);
 });
+
+/* *
+ *
+ *  API Declarations
+ *
+ * */
+
+/**
+ * Gets fired after closing the fullscreen
+ *
+ * @callback Highcharts.FullScreenAfterFullscreenCloseCallbackFunction
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart on which the event occured.
+ *
+ * @param {global.Event} event
+ *        The event that occured.
+ */
+
+/**
+ * Gets fired before closing the fullscreen
+ *
+ * @callback Highcharts.FullScreenBeforeFullscreenCloseCallbackFunction
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart on which the event occured.
+ *
+ * @param {global.Event} event
+ *        The event that occured.
+ */
+
+/**
+ * Gets fired after opening the fullscreen
+ *
+ * @callback Highcharts.FullScreenAfterFullscreenOpenCallbackFunction
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart on which the event occured.
+ *
+ * @param {global.Event} event
+ *        The event that occured.
+ */
+
+/**
+ * Gets fired before opening the fullscreen
+ *
+ * @callback Highcharts.FullScreenBeforeFullscreenOpenCallbackFunction
+ *
+ * @param {Highcharts.Chart} chart
+ *        The chart on which the event occured.
+ *
+ * @param {global.Event} event
+ *        The event that occured.
+ */
+
+
+/* *
+ *
+ *  API Options
+ *
+ * */
+
+/**
+ * Fires after fullscreen close
+ *
+ * @sample highcharts/chart/events-fullscreenopen-fullscreenclose/
+ *         Fullscreen title size change
```

Comment:
Could the description be more descriptive? See e.g. API of `chart.events.afterPrint` for a reference

Reply:
@mfilipiec sorry, maybe I missclicked or maybe GitHub is showing it incorrectly.

I meant that the `Fires after fullscreen close` sentence could be more descriptive, not the sample's name.